appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/jmx/MBeanGuageImpl.java

```diff
@@ -47,7 +47,7 @@
  * @author jonathan coustick
  * @since 5.193
  */
-public class MBeanGuageImpl implements Gauge<Long> {
+public class MBeanGuageImpl implements Gauge<Double> {
```

Comment:
Doesn't this mean you change the type of all MBean gauges? Or is this just used by the metric you wanted to change?